src/ml/neural_net/tf_activity_classifier_backend.hpp

```diff
@@ -0,0 +1,41 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+*/
+#ifndef UNITY_TOOLKITS_NEURAL_NET_TF_AC_MODEL_HPP_
+#define UNITY_TOOLKITS_NEURAL_NET_TF_AC_MODEL_HPP_
+
+#include <ml/neural_net/model_backend.hpp>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+
+namespace turi {
+namespace neural_net {
+
+namespace py = pybind11;
```

Comment:
Let's avoid using namespace aliases in header files.

https://google.github.io/styleguide/cppguide.html#Aliases